### PR TITLE
tikv-ctl: Add more information for compact and remove-fail-stores

### PR DESCRIPTION
**Pull request by Connor1996**
### What is changed, added or deleted? (Required)

Add more information for compact and remove-fail-stores command

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- Improve https://github.com/tikv/tikv/issues/10040

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by TomShawn** [comment]
@Connor1996 Please involve technical review.

File: tikv-control.md [hunk 1]
```diff
@@ -482,7 +486,7 @@ tikv-ctl --db /path/to/tikv/db unsafe-recover remove-fail-stores -s 4,5 --all-re
 >
 > - 该命令只支持本地模式。在运行成功后，会打印 `success!`。
 > - 一般来说，您需要为指定 Region 的 peers 所在的每个 store 运行此命令。
-> - 如果使用 `--all-regions`，通常需要在集群剩余所有健康的 store 上执行此命令。
+> - 如果使用 `--all-regions`，通常需要在集群剩余所有健康的 store 上执行此命令。需要保证所有目标 store 都停掉服务后进行恢复，否则期间 Region 副本之间的 peer 列表不一致会导致 apply 报错进而引起其他元数据的不一致，最终引发 Region Unavailable 不可用。
```
Comment (BusyJay) [inline tikv-control.md:489]: 只要把调度停了再操作就没这个问题了？

Comment (Connor1996) [reply tikv-control.md:489]: 不行。写入导致 split 也有问题

**Review by BusyJay — APPROVED** [review]

File: tikv-control.md [hunk 2]
```diff
@@ -258,6 +258,10 @@ middle_key_by_approximate_size:
 - `--host` 参数可以指定要 compact 的 TiKV。
 - `-d` 参数可以指定要 compact 的 RocksDB，有 `kv` 和 `raft` 参数值可以选。
 - `--threads` 参数可以指定 compact 的并发数，默认值是 8。一般来说，并发数越大，compact 的速度越快，但是也会对服务造成影响，所以需要根据情况选择合适的并发数。
+- `--bottommost` 参数可以指定 compact 是否包括最下层的文件。有 `default`, `skip` 和 `force` 可选，默认为 `default`。
```
Comment (TomShawn) [inline tikv-control.md:261]:
```suggestion
- `--bottommost` 参数可以指定 compact 是否包括最下层的文件。可选值为 `default`、`skip` 和 `force`，默认为 `default`。
```

File: tikv-control.md [hunk 3]
```diff
@@ -258,6 +258,10 @@ middle_key_by_approximate_size:
...
+    - `default` 表示，只有开启了 compaction filter 时 compact 才会包括最下层文件
```
Comment (TomShawn) [inline tikv-control.md:262]:
```suggestion
    - `default` 表示只有开启了 Compaction Filter 时 compact 才会包括最下层文件。
```

File: tikv-control.md [hunk 4]
```diff
@@ -258,6 +258,10 @@ middle_key_by_approximate_size:
...
+    - `skip` 表示，compact 不包括最下层文件
```
Comment (TomShawn) [inline tikv-control.md:263]:
```suggestion
    - `skip` 表示 compact 不包括最下层文件。
```

File: tikv-control.md [hunk 5]
```diff
@@ -258,6 +258,10 @@ middle_key_by_approximate_size:
...
+    - `force` 表示，compact 总是包括最下层文件
```
Comment (TomShawn) [inline tikv-control.md:264]:
```suggestion
    - `force` 表示 compact 总是包括最下层文件。
```

File: tikv-control.md [hunk 6]
```diff
@@ -482,7 +486,7 @@ tikv-ctl --db /path/to/tikv/db unsafe-recover remove-fail-stores -s 4,5 --all-re
 >
 > - 该命令只支持本地模式。在运行成功后，会打印 `success!`。
 > - 一般来说，您需要为指定 Region 的 peers 所在的每个 store 运行此命令。
-> - 如果使用 `--all-regions`，通常需要在集群剩余所有健康的 store 上执行此命令。
+> - 如果使用 `--all-regions`，通常需要在集群剩余所有健康的 store 上执行此命令。需要保证这些健康的 store 都停掉服务后再进行恢复，否则期间 Region 副本之间的 peer 列表不一致会导致 apply 报错进而引起其他元数据的不一致，最终引发 Region Unavailable 不可用。
```
Comment (TomShawn) [inline tikv-control.md:489]:
```suggestion
> - 如果使用 `--all-regions`，通常需要在集群剩余所有健康的 store 上执行此命令。需要保证这些健康的 store 都停掉服务后再进行恢复，否则期间 Region 副本之间的 peer 列表不一致会导致 Apply 算子报错进而引起其他元数据的不一致，最终导致 Region 不可用。
```

**Review by TomShawn — COMMENTED** [review]
Minor edits

File: tikv-control.md (hunk 6 above)
Comment (Connor1996) [reply tikv-control.md:489]: apply 算子有点奇怪，我们不这么说

Comment (TomShawn) [reply tikv-control.md:489]: 确实有这种说法。“apply” 没有单位没有修饰词，不了解实现原理的读者看到还以为是个自然语言单词“apply“，”应用，适用”的意思？
image

Comment (Connor1996) [reply tikv-control.md:489]: 这里不是 apply 算子的意思，换了个说法麻烦再看下

File: tikv-control.md [hunk 7]
```diff
@@ -482,7 +486,7 @@ tikv-ctl --db /path/to/tikv/db unsafe-recover remove-fail-stores -s 4,5 --all-re
 >
 > - 该命令只支持本地模式。在运行成功后，会打印 `success!`。
 > - 一般来说，您需要为指定 Region 的 peers 所在的每个 store 运行此命令。
-> - 如果使用 `--all-regions`，通常需要在集群剩余所有健康的 store 上执行此命令。
+> - 如果使用 `--all-regions`，通常需要在集群剩余所有健康的 store 上执行此命令。需要保证这些健康的 store 都停掉服务后再进行恢复，否则期间 Region 副本之间的 peer 列表不一致会导致执行 split 或者 remove-peer 时报错进而引起其他元数据的不一致，最终引发 Region 不可用。
```
Comment (TomShawn) [inline tikv-control.md:489]:
```suggestion
> - 如果使用 `--all-regions`，通常需要在集群剩余所有健康的 store 上执行此命令。需要保证这些健康的 store 都停掉服务后再进行恢复，否则期间 Region 副本之间的 peer 列表不一致会导致执行 `split-region` 或者 `remove-peer` 时报错进而引起其他元数据的不一致，最终引发 Region 不可用。
```

**Pull request merged**